tools/mlsstats/exec/Main.hs

```diff
@@ -0,0 +1,36 @@
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2022 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module Main
+  ( main,
+  )
+where
+
+import Imports
+import MlsStats.Options
+import MlsStats.Run (run)
+import Options.Applicative
+
+main :: IO ()
+main = do
+  opts <- execParser (info (helper <*> optsParser) desc)
+  run opts
+  where
+    desc =
+      header "mlsstats"
+        <> progDesc "MlsStats - Proteus to MLS migration statistics for admins"
```

Comment:
```suggestion
        <> progDesc "MLS Stats - Proteus to MLS migration statistics for admins"
```

or it should be `mlsstats`, `MlsStats` is neither the name of the binary nor how a user will see it as.